s3transfer/manager.py

```diff
@@ -27,26 +29,40 @@
 class TransferConfig(object):
     def __init__(self,
                  multipart_threshold=8 * MB,
-                 max_concurrency=10,
                  multipart_chunksize=8 * MB,
-                 max_queue_size=0,
+                 max_request_concurrency=10,
```

Comment:
To clarify I was asking specifically how you plan on mapping these.  Is max_concurrency -> max_request_concurrency?  Are you not going to allow users to specify max submission concurrency in the existing boto3 transfer config?